test/e2e/storage/testsuites/subpath.go

```diff
@@ -315,7 +315,7 @@ func testSubPath(input *subPathTestInput) {
 	})
 
 	It("should unmount if pod is force deleted while kubelet is down [Disruptive][Slow]", func() {
-		if input.volType == "hostPath" || input.volType == "hostPathSymlink" {
+		if strings.HasPrefix(input.volType, "hostPath") || strings.HasPrefix(input.volType, "csi-hostpath") {
```

Comment:
Can this be done without hard-coding the names?

Third-party drivers will be named differently and may also have to skip this test.

Reply:
@pohly 

Thank you for pointing it out. I will fix it.

@msau42 

Could you explain the reason why this subPath test should be skipped for hostPath, so that I can name the flag in ```TestDriver``` to skip this test properly?
Is it due to non-attachable, local, or lack of reconstruction code?

Reply:
It's a regression due to subpath fixes we made a while back: https://github.com/kubernetes/kubernetes/issues/61446.  It's not intended for some drivers to not support reconstruction and others to support it, so I'm not sure if we really want to make it a parameter.

Reply:
@msau42 

Thank you for your explanation. I will add a comment there to mark it as TODO with a link to the issue.